src/app/activityintro/activityintro.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-activityintro',
+  templateUrl: './activityintro.component.html',
+  styleUrls: ['./activityintro.component.scss']
+})
+
+
+export class ActivityintroComponent {
+  public position = 'col-md-10 col-md-offset-2 introbody text-center';
+
+  constructor() { }
+
+  toggle() {
```

Comment:
The method which I am using requires the position string to contain the component specific css classes.  There are many different methods for inter component interaction but this one was the simplest.